test/sequence-steps/ask-prerelease-identifier.js

```diff
@@ -1,34 +0,0 @@
-import test from "ava";
```

Comment:
so nice to see these tests deleted!

was a failed experiment... just like coffeescript was when i introduced it in a project years ago. the more you know!

hopefully jest doesn't end up the same way